sentry-sample/src/main/res/layout/activity_main.xml

```diff
@@ -1,43 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:tools="http://schemas.android.com/tools"
```

Comment:
What changed here? Not sure what to review

Reply:
probably changed by the IDE automatically and will be reverted, Draft PR